vllm_ascend/quantization/w8a8_dynamic.py

```diff
@@ -152,7 +157,11 @@ def fused_experts_with_mc2(
     if quant_mode == 0:
         dynamic_scale = None
 
-    down_out_list = apply_mlp(expand_x,
+    # place hidden_states in a list to transfer its ownership into the `apply_mlp` function
```

Comment:
I'm fine with this quick fix.  you have describe the case more clear later. For example, this `copy to list` action aims to deal with the lifecycle for the tensor so that the memory usage can be improved.